app/api/auth/[...nextauth]/options.ts

```diff
@@ -0,0 +1,31 @@
+import { NextAuthOptions } from "next-auth";
+import CredentialsProvider from "next-auth/providers/credentials";
+import { SupabaseAdapter } from "@auth/supabase-adapter";
```

Comment:
maybe remove the supabase stuff for now actually until you have the basic Creds provider working, seems like you aren't using it anyway